data/json/snippets/health_msgs.json

```diff
@@ -187,6 +187,6 @@
   {
     "type": "snippet",
     "category": "low_cal",
-    "text": "You feel that your body needs more nutritious food."
+    "text": "You feel that you need to eat more calory dense food."
```

Comment:
```suggestion
    "text": "You feel that you need to eat more calorie dense food."
```

Reply:
It should be hyphenated, like "more calorie-dense food."